dapp/config/qa.json

```diff
@@ -1,7 +1,19 @@
 {
   "env": "qa",
   "web3": {
-    "fuseProvider": "https://rpc.fuse.io"
+    "fuseProvider": "https://rpc.fuse.io",
+    "bridge": {
+      "addresses": {
+        "homeToForeign": {
+          "fuse": {
+            "MultiBridgeMediator": "0x6A90210fc7643Fd4E829204D526C060f0fc347D6"
```

Comment:
@LiorAgnin not needed here, cause when it's not in `qa.json` it takes the default